### ISSUE TITLE
CI testing of commit 10f5716 fails in Test "STFL and STFLE"

### ISSUE THREAD

**Issue by wfjm** [SDL-Hercules-390/hyperion#170]
I've updated the Travis CI testing branch of my fork to commit 10f5716, see #143. `make check` fails for all tested compilers (now 4 gcc and 2 clang versions) with
```
>>>>> line 88108: Storage compare mismatch.
        Want: R:0000000000000200 F3F4FFFB FCF5C800 00000000 00000000  "First two doublewords of truncated facilities list"
        Got:  R:0000000000000200 F3F4FFFB FCFDC800 00000000 00000000
>>>>> line 88112: Storage compare mismatch.
        Want: R:0000000000000210 F3F4FFFB FCF5C800 201C0000 00000000  "First two doublewords of extended facilities list"
        Got:  R:0000000000000210 F3F4FFFB FCFDC800 201C0000 00000000
Test "STFL and STFLE":  10 OK compares.  2 failures.
```

See [Travis build 474805415](https://travis-ci.org/wfjm/hyperion/builds/474805415) for full report.

Whether this is a genuine Hercules issue or yet another `regina-rexx` flaw I don't know.

**Comment by Fish-Git** [comment]
Fixed by commit 244b5f0fc63d240f681e87d8ef48d5f1caad5cf3.

Thank you for reporting this, Walter!

**Comment by Fish-Git** [comment]
> Whether this is a genuine Hercules issue or yet another `regina-rexx` flaw I don't know.

It was a simple oversight in one of our test cases and had nothing to do with Regina Rexx.

One of the z/Architecture facilities was recently changed to always be enabled by default, whereas before it wasn't (you had to manually enable it). As a result, the "STFL and STFLE" test (Store Facilities List and Store Facilities List Extended) was failing because Facilities List bit 44 was now on by default, whereas before it wasn't (second byte of the second word now being `FD` instead of the `F5` it was expecting). The fix was to just change the test's "Want" condition to `FD` since the facility is now always enabled (the bit is always on) by default.
